requirements.txt

```diff
@@ -1,6 +1,6 @@
-matplotlib==3.3.2
-pandas==1.1.2
-importlib-metadata==2.0.0
-zipp==2.2.0 # hard requirement of bzt 1.15.0
-bzt==1.15.0
-locust==1.2.3
\ No newline at end of file
+matplotlib==3.3.3
+pandas==1.1.4
+importlib-metadata==3.1.1
+zipp==3.4.0
```

Comment:
Why do we need this?

Reply:
Just have checked and ensured that we don't need it here